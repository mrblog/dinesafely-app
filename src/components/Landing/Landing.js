import React from 'react';
import { Places } from '../Places'; 

const Landing = () => (
    <div>
        <h1>Check and grade the COVID-19 Safety Measures of your favorite places</h1>
        <Places/>
    </div>
)

export default Landing; 