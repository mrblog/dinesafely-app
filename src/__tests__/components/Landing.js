import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson  } from 'enzyme-to-json';

jest.mock("react-router-dom", () => ({
    ...jest.requireActual("react-router-dom"),
    useLocation: () => ({
        pathname: "localhost:3000/"
    })
}));

import Landing from '../../components/Landing/Landing'

it('renders correctly when there are no parameters', () => {
    Enzyme.configure({ adapter: new Adapter() });

    const wrapper = shallow(<Landing />)

    const tree = renderer.create(shallow(<Landing />)).toJSON();
    expect(shallowToJson(wrapper)).toMatchSnapshot();
});