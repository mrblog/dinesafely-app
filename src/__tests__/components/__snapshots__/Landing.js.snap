// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no parameters 1`] = `
<div>
  <h1>
    Check and grade the COVID-19 Safety Measures of your favorite places
  </h1>
  <Places />
</div>
`;
